Flea/Pages/CustomerOverview.razor

```diff
@@ -0,0 +1,95 @@
+@page "/CustomerOverview"
+<div>
+    <div class="view-head">
+        <input class="search-input" type="search" value="" placeholder="Søg efter kunde" @oninput="Searchbar">
+        <button class="button button--outline create-button" @onclick="NewCustomer">Tilføj Ny Loppe</button>
+    </div>
+
+    @foreach (var customer in ShownCustomers)
+    {
+        <div>
+            <UserListItem Id="@customer.Id" Name="@customer.Name" TeleNumber="@customer.TeleNumber"
+                          OnDelete="@DeleteCustomerInformation" OnEdit="@EditCustomerInformation">
+            </UserListItem>
+        </div>
+    }
+</div>
+
+@code
+{
+    private string _search = "";
+
+    // Dummy data ---
+    class Customer
+    {
+        public int Id { get; set; }
+        public string Name { get; set; }
+        public string TeleNumber { get; set; }
+    }
+
+    private List<Customer> Customers = new()
+    {
+        new Customer() {Id = 1, Name = "Sebastian Aaholm", TeleNumber = "24823423"},
+        new Customer() {Id = 2, Name = "Jens Jensen", TeleNumber = "4541341"},
+        new Customer() {Id = 3, Name = "Hans Nielsen", TeleNumber = "8888888"},
+    };
+
+    // ---
+
+    private IEnumerable<Customer> ShownCustomers;
```

Comment:
Private fields should be written in camelCase with a prefixed `_`.  
Example: `_shownCustomers`.